OOVPADatabase/D3D8/3911.inl

```diff
@@ -24,44 +24,45 @@
 // ******************************************************************
 
 // ******************************************************************
-// * D3DDevice__ManualFindEventHandleGeneric
+// * D3DDevice__m_VerticalBlankEvent__ManualFindGeneric
```

Comment:
Feels like these changes should be split into a separate commit or PR.
It's not described in the PR description and hides the other changes.

Reply:
Most of this seems to be in PR 96 now.
This PR still needs a temporary rebase so the commit loses that information.

Reply:
Yeah, I'm busy with that

Reply:
Done. I'll move the rest of the changes in separate commits / PR's as well.